pyproject.toml

```diff
@@ -10,16 +10,25 @@ Django = "^4.1.3"
 djangorestframework = "^3.14.0"
 mysqlclient = "^2.1.1"
 django-webpack-loader = "^1.7.0"
+requests = "^2.28.1"
```

Comment:
When executing `make mypy` it wants to have `types-requests` in addition to that.